cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -634,13 +634,15 @@ recompiled and re-deployed.)
    - Using Yum (on one line)::
 
        $ sudo yum install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
```

Comment:
So, ``cdap-hbase-compat-1.0-cdh`` gets installed even if you are running on HDP?